### PR TITLE
1060 logs

### PR DESCRIPTION
**Pull request by peter-lyons**
# Issue Addressed
This PR closes the update weblogs and airflow logs tasks

# Description
Logs now write to /data/logs

# Tests
Deployed on logs branch in dev installation on flashcrow. Observed new log files being written to the specified directory.

### CODE REVIEW

File: deploy_scripts/install.sh [hunk 1]
```diff
@@ -100,3 +100,6 @@ set -u
 
 # link DAGs
 ln -s /home/ec2-user/move_etl/scripts/airflow/dags /home/ec2-user/airflow/dags
+
+# make log directory
+mkdir -p /home/ec2-user/data/log/etl
```
Comment (candu) [inline deploy_scripts/install.sh:105]: (we're actually using `/data/log/etl`)

also - we might want to create both `/data/log/etl` and `/data/log/web`, but that can be changed later (e.g. when we go to set up automated Airflow deployments)

**Review by candu — APPROVED** [review]
LGTM (but one note about the log directory for later follow-up)

**Comment by leonL** [comment]
Looks good to me.

**Review by leonL — COMMENTED** [review]
Looks good to me.

**Pull request merged**